zulip_bots/zulip_bots/bots/youtube/youtube.py

```diff
@@ -24,6 +25,14 @@ def usage(self):
 
     def initialize(self, bot_handler):
         self.config_info = bot_handler.get_config_info('youtube')
+        # Check if API key is valid. If it is not valid, don't run the bot.
+        try:
+            search_youtube('test', self.config_info['key'])
+        except HTTPError as e:
+            if (e.response.json()['error']['errors'][0]['reason'] == 'keyInvalid'):
```

Comment:
Not sure if this is a right approach to  check the API key.  Seeking advice as to how to proceed or if this approach is fine